docs/01-getting-started.md

```diff
@@ -13,32 +13,32 @@ Before running this demo functionality, you will need the following installed:
 
 * [Bash](https://www.gnu.org/software/bash/): Tested with v3.2. Use `bash --version`.
 * [Docker](https://docs.docker.com/get-docker/): Tested with v20.10.17
-* [Java 11/17](https://openjdk.java.net/install/)
-* [Maven](https://maven.apache.org/): Tested with v3.8.6
 
 ### Deployment environment
 
 You can use the AWS CDK to create an EC2 instance in a VPC that is suitable for deploying Sleeper. A local Docker image
-contains the necessary dependencies and scripts to do this. Run these commands in the directory `scripts/local` to build
-the Docker image and deploy the environment:
+contains the necessary dependencies and scripts to do this. Run these commands to install a CLI for working with the
+local Docker image:
 
 ```bash
-./build.sh
-./runInDocker.sh aws configure
-./runInDocker.sh cdk bootstrap
-./runInDocker.sh environment deploy TestEnvironment
+curl "https://raw.githubusercontent.com/gchq/sleeper/[version]/scripts/local/install.sh" -o ./sleeper-install.sh
+chmod +x ./sleeper-install.sh
+./sleeper-install.sh [version]
+sleeper aws configure
+sleeper cdk bootstrap
```

Comment:
Should we comment that this only needs doing once per account?

Reply:
Will do. I don't think it causes a problem if it's already been done?

Reply:
Done